packages/immutable-cdn/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.0] - 2022-01-13
+
+Initial release
```

Comment:
Would be nice to see what is included in the initial release, makes future changes more referenceable to the original state